Algorithm-Check-for-Palindromes.md

```diff
@@ -1,6 +1,8 @@
+# Algorithm Check for Palindromes
+
 ![](http://i.imgur.com/ozDWKEi.jpg)
 
-# Explanation:
+## Explanation:

```

Comment:
The Explanation should appear in chat at the first call. With `h2`, it won't appear.